src/subscriptions/subscriptions.service.ts

```diff
@@ -262,15 +261,4 @@ export class SubscriptionsService {
 
     this.scheduleSendHandshake(id);
   }
-
-  // TODO @polymath-eric: implement random secret generation
```

Comment:
I think this was an inherited TODO. The crypto package should give sufficiently random bytes.

This is moved to util to make API keys